aodntools/timeseries_products/Documentation/Hourly_timeseries.md

```diff
@@ -0,0 +1,157 @@
+
+Additional metadata variables (ancillary variables) will be included, derived from the binning process and the source files: 
+`VoI_std`: standard deviation per averaging bin;
+`VoI_min`: minimum value per bin;
+`VoI_max`: maximum value per bin;
+`VoI_count`: number of observations in bin;
+
+In order to keep track of the provenance of VoIs in the aggregated file, accessory variables are created:
+
+- `instrument_index(OBSERVATION)`: index [0:number of files] of the instrument used, referencing the INSTRUMENT dimension.
+- `source_file(INSTRUMENT, string256)`: URLs of the files used
+- `instrument_id(INSTRUMENT, string256)`: concatenated deployment_code, instrument and instrument_serial_number from the global attributes of each file
+- `LATITUDE(INSTRUMENT)`: LATITUDE per instrument.
+- `LONGITUDE(INSTRUMENT)`: LONGITUDE per instrument.
+- `NOMINAL_DEPTH(INSTRUMENT)`: nominal depth per instrument, from the input file’s variable NOMINAL_DEPTH or global attribute instrument_nominal_depth.
+
+The global metadata will be a set of IMOS standard attributes, and specific standard attributes concatenated from input file attributes
+
+Product files will be made available via the AODN THREDDS server, and regularly updated to include new data.
+
+### Attributes
+
+The variable attributes will comply with the IMOS metadata standards.
+
+The global metadata will be a set of IMOS standard attributes. Fixed attributes are read from a JSON file that contains the {key:value} pairs for each of them. See the contents of this file at the end of this document. 
+
+Attributes specific to each aggregated product, are added as follows:
+
+- `site_code`: obtained from the input files (should be the same in all of them);
+- `time_coverage_start`, `time_coverage_end`: set to the full range of TIME values in the aggregated file;
+- `geospatial_vertical_min/max`: set to the full range of DEPTH values in the aggregated file;
+- `geospatial_lat_min/max`: set to the full range of LATITUDE values in the aggregated file;
+- `geospatial_lon_min/max`: set to the full range of LONGITUDE values in the aggregated file;
+- `date_created`: set to the date/time the product file is created;
+- `history`: set to “<date_created>: Aggregated file created.”;
+- `keywords`: set to a comma-separated list of the main variable names (“<VoI>, TIME, DEPTH, LATITUDE, LONGITUDE”);
+- `lineage`: a statement about how the file was created, including a link to the code used, and any input parameters (except the input files, which are listed in the source_file variable)
+- `title`: "Long time series Hourly Aggregated product: all available non-velocity variables at <site_code> between <time_coverage_start> and <time_coverage_end>"
+- `values_retained`: indicates what values are used according to its qc control flags
```

Comment:
We've renamed this to `included_values_flagged_as`.
This might be a clearer description: "a list of the quality-control flags accepted for inclusion into the binning"?